cypress/e2e/lpa/cases/documents.cy.js

```diff
@@ -0,0 +1,50 @@
+describe("Documents", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.intercept({ method: "GET", url: "/*/v1/persons/*/cases" }).as(
+      "casesRequest"
+    );
+
+    cy.loginAs("LPA Manager");
+    cy.createDonor().then(({ id: donorId }) => {
+      cy.createLpa(donorId).then(({ id: lpaId }) => {
+        cy.wrap(donorId).as("donorId");
+        cy.wrap(lpaId).as("lpaId");
+      });
+    });
+  });
+
+  it("should create a draft with an insert", function () {
+    cy.visit(`/lpa/#/person/${this.donorId}/${this.lpaId}`);
+    cy.wait("@casesRequest");
+
+    cy.get("#CreateDocument").click();
+
+    cy.frameLoaded(".action-widget-content iframe");
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().find("#f-templateId").type("IT-AT-LPA");
+      getBody().find(".autocomplete__menu").contains("IT-AT-LPA: Letter to attorney - LPA").click();
+      getBody().find("button[type=submit]").click();
+    });
+
+    cy.wait(1500);
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().contains("Select document inserts");
+      getBody().find( "#f-IT-11-all").click();
+      getBody().find("button[type=submit]").click();
+    });
+
+    cy.wait(1500);
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().contains("Select a recipient");
+      getBody().contains("label", "Bob Sponge");
+      getBody().find( "[data-module='recipient-checkbox']").click();
+      getBody().contains("button", "Create draft document").click();
+    });
+
+    cy.wait(3000);
+    cy.enter(".action-widget-content iframe").then((getBody) => {
+      getBody().contains("textarea", "Dear Sponge");
+      getBody().contains("textarea", "explanations of any transactions that are not detailed in their financial records");
+    });
```

Comment:
I needed to add more time to wait for the edit draft screen to load, i found 3 seconds to be reliable.. is this going to add a lot of time to the test run in the build?

Reply:
Yeah not optimal but know from experience how horrible iframes+Cypress are. The tests are run in parallel, so it shouldn't be too much of a problem, but if it becomes an issue then we could look at opening the iframe in a new tab so that we don't need the implicit waits.